### PR TITLE
Corrected Permissions Table seeding

### PR DESCRIPTION
**Pull request by srivastava-jyoti**
Issue: On running ` php artisan db:seed --class=PermissionsTableSeeder` previous values were again seeded in the database.

### CODE REVIEW

File: database/Seeders/PermissionsTableSeeder.php [hunk 1]
```diff
@@ -15,8 +16,10 @@ class PermissionsTableSeeder extends Seeder
      */
     public function run()
     {
-        \DB::table('permissions')->delete();
+        DB::statement('SET FOREIGN_KEY_CHECKS=0;');
+        \DB::table('permissions')->truncate();
```
Comment (GauravGusain98) [inline database/Seeders/PermissionsTableSeeder.php:20]: I think let's not solve this issue in this manner. AS if we delete the permissions and again create new ones (the permission will have new ids) then the roles cannot find the permissions to which they are mapped as we have deleted those permissions and now the id to which roles are mapped does not exists.

For this we can use the `createOrUpdate()` function of laravel ORM. 
[Reference documentation](https://laravel.com/docs/8.x/eloquent#upserts)

Comment (GauravGusain98) [reply database/Seeders/PermissionsTableSeeder.php:20]: Also then we have to remove the delete statements from the seeder.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

File: database/Seeders/PermissionsTableSeeder.php (hunk 1 above)
Comment (srivastava-jyoti) [reply database/Seeders/PermissionsTableSeeder.php]: Okay, I'll correct this.

File: database/Seeders/PermissionsTableSeeder.php [hunk 2]
```diff
@@ -18,104 +19,105 @@ public function run()
         \DB::table('permissions')->delete();
```
Comment (GauravGusain98) [inline database/Seeders/PermissionsTableSeeder.php:19]: This is not required. Please remove this line

File: database/Seeders/PermissionsTableSeeder.php [hunk 3]
```diff
@@ -18,104 +19,105 @@ public function run()
...
         Artisan::call('permission:cache-reset');
```
Comment (GauravGusain98) [inline database/Seeders/PermissionsTableSeeder.php:20]: Same here. This is also not required.

**Review by GauravGusain98 — APPROVED** [review]
Looks good.

**Pull request merged**